test/functional/net.py

```diff
@@ -32,6 +32,39 @@ def setup_network(self):
         self.sync_all()
 
     def run_test(self):
+        self._test_connection_count()
+        self._test_getnettotals()
+        self._test_getnetworkinginfo()
+        self._test_getaddednodeinfo()
+
+    def _test_connection_count(self):
+        # connect_nodes_bi connects each node to the other
+        assert_equal(self.nodes[0].getconnectioncount(), 2)
+
+    def _test_getnettotals(self):
+        # check that getnettotals totalbytesrecv and totalbytessent
+        # are consistent with getpeerinfo
+        peer_info = self.nodes[0].getpeerinfo()
+        assert_equal(len(peer_info), 2)
+        net_totals = self.nodes[0].getnettotals()
+        assert_equal(sum([peer['bytesrecv'] for peer in peer_info]),
+                     net_totals['totalbytesrecv'])
```

Comment:
nit: personally I'd prefer if it was on a single line, such that it shows up as a whole in case of a print traceback.